src/tests/ftest/dfuse/pil4dfs_fio.yaml

```diff
@@ -31,11 +31,9 @@ server_config:
 
 pool:
   size: 90%
-  control_method: dmg
 
 container:
   type: POSIX
-  control_method: daos
```

Comment:
Ah! This one for the container SHOULD stay

Reply:
- [ ] Restore missing contro_method for container

Reply:
Fixed with commit https://github.com/daos-stack/daos/pull/13797/commits/69781fdc9e97f9cf2602f3ce2b2447c22ff3fdf8